From: Stuart Cardall <developer@it-offshore.co.uk>
Date: Fri, 10 Feb 2017 15:26:30 +0000
Subject: [PATCH] add contrib/runit.run

Adds example run script for service supervision with runit.

Alpine notes:
* This patch is ported from upstream commit:
  https://github.com/wojtekka/6tunnel/commit/f800eb1fe5b54ad1022c1bebd2a41a650b86752d

new file mode 100755
index 0000000..d87789e
--- /dev/null
+++ b/contrib/runit.run
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# example run script for service supervision with runit
+# http://smarden.org/runit/
+#
+# redirect ipv4 port 80 to internal ipv6 port 80 & drop privileges
+#
+exec 2>&1
+exec 6tunnel -u nobody -d -p /run/80.pid 80 a:dead:beef:1 80
+
+
